pages/RF-CTF-RFID-NFC-Challenges.md

```diff
@@ -25,60 +29,103 @@ Industrial fixed position readers from Impinj or Alien also work well with exter
 
 UHF RFID Modules are another potential option for integrating into your kit:
 - https://www.sparkfun.com/products/14066
-- https://shop.m5stack.com/products/uhf-rfid-unit-jrd-4035
 
 Additionally there are some (currently untested) SDR based UHF RFID reader implementations:
 - https://github.com/AdamLaurie/Gen2-UHF-RFID-Reader
 
 # Challenges
 
-## RFHS_RFID_PACS_X
-### Physical
-This time we didn't just bring a building in a box, we brought the parking lot as well. There will be a range of different credential technlogies (LF, HF, UHF), some you can get your hands on, and some placed out of reach requiring creativity and determination.  You will need to use a variety of tools (including a UHF RFID Reader/Writer) and techniques to interact with our PACS (Physical Access Control System).
+## Physical Access Control System (PACS) Box
+We didn't just bring a building in a box, we brought the parking lot as well. There will be a range of different credential technlogies (LF, HF, UHF), some you can get your hands on, and some placed out of reach requiring creativity and determination.  You will need to use a variety of tools (including a UHF RFID Reader/Writer) and techniques to interact with our access control system.
 
-5 sets of cards will be at the front table
+### Rules
+- Sets of cards will be available at the front table and spread out on a few contestant tables.
+- You can interact with the badges, you can interact with the readers, but you can't take the badges to the readers.
+- Do not attempt to reconfigure the readers.
```

Comment:
This doesn't say successfully reconfiguring the reader isn't allowed, it just says attempts are...

Reply:
merge it after fixing this, or without fixing this if you want people to reconfigure the reader